### DISCUSSION TITLE
Scripts - Description Mouse Over

### DISCUSSION THREAD

**Discussion by kbvc** [dbeaver/dbeaver#13194] [Q&A] [answered]
Project Tab: 

My lists of scripts is growing exponentially and the titles are looking similar and have less impact when searching.

Adding a description to the scripts in a mouseover message box would help to describe them before opening them thus making the search process more efficient.

Alternatively, an expansion selector could be added to open up a description.

**Comment by uslss** [comment, marked as answer]
Please, uncheck Show file contents in tooltip option in preferences
![image](https://user-images.githubusercontent.com/31996417/125592676-d4577316-4e0e-4bdb-ae1c-0b1caf85e6ed.png)

**Comment by kbvc** [comment]
I did not know this existed and it works great!  It was unchecked by default so I checked it.  Thank you for the quick response!
